### ISSUE TITLE
Resource exhausted ERROR

### ISSUE THREAD

**Issue by hemanth2410** [Sentdex/pygta5#98]
while training with Tensorflow GPU  i am getting resource exhausted error
![error](https://user-images.githubusercontent.com/24288422/39032680-2d530fe2-448c-11e8-98a0-c1b3fef70cd5.png)

**Comment by hemanth2410** [comment]
PC specs 
Intel core i7 7700HQ @ 3.4 GHz
8 GB SODIMM DDR 4 RAM
GTX 1050 4GB(NETBOOK)

**Comment by kunakl07** [comment]
What is the command that you used for training ?

**Comment by Denrose11** [comment]
> if it's not problem in path to files, you can try add batch_size=32 or 16 or even 4 (by default its 64) to model.fit() . Neural networks are really heavy for gpu memory, so 16 worked for me, problem is, it may cause some acc and loss issues on later epochs, but without this can't run at all, so i'm curious does it affects a lot, I mean, will model be okay with such size, or its bad idea.

how to decrease the batch size in model.fit() for version .03
